packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -0,0 +1,29 @@
+import { UserAgentApplication as AzureActiveDirectory } from 'msal'
+
+export type { AzureActiveDirectory }
+import type { AuthClient } from './'
+
+export type AzureActiveDirectoryClient = AzureActiveDirectory
+export interface AzureActiveDirectoryUser {}
+
+export const azureActiveDirectory = (client: AzureActiveDirectoryClient): AuthClient => {
+  return {
+    type: 'azureActiveDirectory',
+    client,
+    restoreAuthState: async () => {},
+    login: async (options?) => {
+      await client.loginPopup(options)
+    },
+    logout: (options?) => client.logout(options),
+    signup: async (options?) => {
+      await client.loginPopup(options)
+    },
```

Comment:
```suggestion
    signup: async (options?) => await client.loginPopup(options),
```

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/f14a58c31124662ac9df1e715d78cdeea5d14262